app/routes/index.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  afterModel() {
+    this.transitionTo('reminders');
```

Comment:
Also just a comment but `transitionTo` will keep the '/' path in the browser history. If you never need to USE that path (like in no situation would a user hit the "back" button to get to the root page), i'd use `replaceWith` instead